SCClassLibrary/Common/Control/Server.sc

```diff
@@ -1018,7 +1018,7 @@ Server {
 				this.prOnServerProcessExit(exitCode);
 			});
 			("Booting server '%' on address %:%.").format(this.name, addr.hostname, addr.port.asString).postln;
-			if(options.protocol == \tcp, { addr.tryConnectTCP(onComplete) }, onComplete);
+			if(options.protocol == \tcp, { addr.tryConnectTCP(onComplete, maxAttempts: 20) }, onComplete);
```

Comment:
FYI it's ever so slightly better for memory and performance if you don't use keyword arguments -- if you wrote `tryConnectTCP(onComplete, nil, 20)` here it would save 2 bytecodes, save you from storing a symbol literal in the stack frame, and also not require a runtime symbol table lookup (since the interpreter cannot verify at compile time whether or not a given argument name is present in a method). these are of course very small things, and this function is definitely not worth optimizing, but i don't think many people writing SC are aware of this, so just wanted to make the point; it does add up over time.

Reply:
Thanks for pointing this out! It's good to present optimal code at least in the class library, and I certainly don't think about this too often. I will make this change.

Reply:
I've removed the keyword argument :)